client/web/src/store.js

```diff
@@ -3,13 +3,15 @@ import thunkMiddleware from "redux-thunk";
 import { authentication } from "./Structure/Authentication"
 import {topics} from './Structure/Topics/State/reducer'
 import { admin } from './Structure/Admin'
+import {questionState} from './Structure/Questions/State/reducer'
 
```

Comment:
use the index.js to import, Don't use entire file path `./Structure/Questions/State/reducer` and change the name from questionState to other